libsolidity/analysis/TypeChecker.cpp

```diff
@@ -699,6 +699,9 @@ bool TypeChecker::visit(InlineAssembly const& _inlineAssembly)
 				}
 			}
 
+			if (dynamic_cast<FixedPointType const*>(var->type()))
+				solAssert(false, "FixedPointType not implemented.");
```

Comment:
Maybe only this?
```suggestion
			solAssert(dynamic_cast<FixedPointType const*>(var->type()), "FixedPointType not implemented.");
```

Reply:
I guess this has already been addressed. Logic here should be `!dynamic_cast<...>` and not `dynamic_cast<...>`